pkg/products/rhsso/reconciler.go

```diff
@@ -106,23 +108,19 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 	ns := r.Config.GetNamespace()
 
 	phase, err := r.ReconcileFinalizer(ctx, serverClient, installation, string(r.Config.GetProductName()), func() (integreatlyv1alpha1.StatusPhase, error) {
-		// Check if namespace is still present before trying to delete it resources
-		_, err := resources.GetNS(ctx, ns, serverClient)
-		if !k8serr.IsNotFound(err) {
```

Comment:
Un-installation gets stuck here.
I think this check for the existence of the namespace should be here. What was the motivation for removal?

Reply:
this wasn't removed, perhaps it was some merging conflict